bodhi/server/ffmarkdown.py

```diff
@@ -34,30 +38,29 @@
 BUGZILLA_RE = r'([a-zA-Z]+)(#[0-9]{5,})'
 
 
-def user_url(name):
+def user_url(name: str) -> str:
     """
     Return a URL to the given username.
 
     Args:
-        name (basestring): The username of the user we want a URL for.
+        name: The username of the user we want a URL for.
     Returns:
-        basestring: A URL to the requested user.
+        A URL to the requested user.
     """
     request = pyramid.threadlocal.get_current_request()
     return request.route_url('user', name=name)
 
 
-def bug_url(tracker, idx):
+def bug_url(tracker: str, idx: typing.Union[int, str]) -> typing.Optional[str]:
     """
     Return the URL for the given bug.
 
     Args:
-        tracker (basestring): Which bug tracker is being referenced. May be any of 'fedora',
-            'gcc', 'gnome', 'kde', 'mozilla', 'pear', 'perl', 'php', 'python', 'rh', 'rhbz'
-            or 'sourceware'.
-        idx (basestring or int): The bug number.
+        tracker: Which bug tracker is being referenced. May be any of 'fedora',
+            'gnome', 'kde', 'mozilla', 'pear', 'perl', 'php', 'python', 'rh', or 'rhbz'.
```

Comment:
It looks like gcc and sourceware got dropped from this list, possibly during a rebase?